documentation/Language/Locale.md

```diff
@@ -0,0 +1,49 @@
+# Locale
+
+This feature of the language allows specifing localization tokens and values for languages indentified by [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) codes.
+
+```mint
+locale en {
+  ui: {
+    buttons: {
+      ok: "OK"
+    }
+  }
+}
+
+locale hu {
+  ui: {
+    buttons: {
+      ok: "Rendben"
+    }
+  }
+}
+```
+
+A locale consists of a tree structure where the keys are lowercase identifiers and the values are expressions.
+
+To localize a value you need to use the locale token:
+
+```
+:ui.buttons.ok
+```
+
+or if it's a function if can be called:
+
+```
+:ui.buttons.ok(param1, param2)
```

Comment:
```suggestion
locale en {
  ui: {
    buttons: {
      ok: (param1 : String, param2 : String) { 
        "Button #{param1} #{param2}!" 
      }
    }
  }
}

:ui.buttons.ok(param1, param2)
```


May be worth adding an example of a function being used in the documentation for the visual learners :smile: